src/get-started/localisation/index.md.njk

```diff
@@ -0,0 +1,92 @@
+---
+layout: layout-pane.njk
+title: Using localisation with GOV.UK Frontend
+section: Get started
+theme: How to guides
+order: 5
+description: Localisation allows you to translate GOV.UK Frontend components into other languages
+---
+
+Localisation allows you to translate GOV.UK Frontend components into other languages
+
+## Localisation using Nunjucks
+
+If you’re using the Nunjucks macros provided by GOV.UK Frontend, you can localise the UI text that appears within components by passing translation strings as named parameters.
+
+We have provided specific localisation guidance for some of the components. These include:
+
+* [accordion](/components/accordion/)
+* [character count](/components/character-count/)
+* [error message](/components/error-message/)
+* [footer](/components/footer/)
+* [header](/components/header/)
+* [summary list](/components/summary-list/)
+
```

Comment:
Some of the specific guidance is about localisation with JavaScript, so I wonder if we should split this list or move it so it's not under the Nunjucks heading? Not necessarily something we need to do now though.